src/main/kotlin/no/nav/syfo/dialogmote/api/domain/DialogmotedeltakerArbeidsgiverVarselDTO.kt

```diff
@@ -10,4 +11,7 @@ data class DialogmotedeltakerArbeidsgiverVarselDTO(
     val lestDato: LocalDateTime?,
     val fritekst: String,
     val document: List<DocumentComponentDTO>,
+    val svarType: DialogmoteSvarType?,
```

Comment:
Burde kanskje være String (slik vi har for andre enums i DTOene, eks. varselType). Samme for arbeidstakerVarsel.